src/tagbar.ts

```diff
@@ -133,6 +133,9 @@ export class TagTool extends Widget {
     });
 
     // Sort the widgets in tag alphabetical order
+    // TODO: find an alternative way to order the tags?
+    // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+    // @ts-ignore
```

Comment:
Looks like there could be an alternative way than using an `undefined` value below?